### ISSUE TITLE
Placeholder linked to Input through aria-describedby

### ISSUE THREAD

**Issue by socproof** [JedWatson/react-select#5651]
Placeholder and Input from react-select are linked not via `label`, `aria-label` or `aria-labeledby`, but through `aria-describedby` - which is absolutely incorrect. 
See the difference: 
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby 
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby

`<div class=" css-1jqq78o-placeholder" id="react-select-11-placeholder">Select...</div>
<div class=" css-19bb58m" data-value="">
<input class="" autocapitalize="none" autocomplete="off" autocorrect="off" id="react-select-11-input" spellcheck="false" tabindex="0" type="text" aria-autocomplete="list" aria-expanded="false" aria-haspopup="true" role="combobox" aria-describedby="react-select-11-placeholder" value="">
</div>`

I believe the current implementation should be redone to make Placeholder linked to Input via `aria-labeledby

**Comment by wbolduc** [comment]
I don't think that using aria-labelledby is the correct choice either here because it has the highest priority as an accessible name overriding the name given by a label for="input-id"

I would prefer using aria-placeholder if the placeholder is ALWAYS plaintext or aria-describedby

There is still a problem though in the current implementation where aria-describedby isn't being respected. To get around this you can use the following custom input 

`const Input = (props: InputProps) => {
	const passed_describedby = props.selectProps["aria-describedby"];
	const describedby =
		props["aria-describedby"] +
		(passed_describedby ? " " + passed_describedby : "");
	return <components.Input {...props} aria-describedby={describedby} />;
};`
Which you can plug into your Select with 
`
<ReactSelect
	components={{
		Input,
	}}
/>`

This works okay because describedby accepts multiple ids. If you are using describedby to identify your errors your resolved description ends up looking like this "Select... Required" Which I think is *okay*

**Comment by emlai** [comment]
We're running into this too. We're trying to link a helper text to react-select via `aria-describedby`, but react-select overrides it with the placeholder.
